gui/notesView.py

```diff
@@ -0,0 +1,41 @@
+import wx
+
+import service
+import gui.globalEvents as GE
+import gui.display as d
+import gui.mainFrame
+
+class NotesView(d.Display):
+    DEFAULT_COLS = []
```

Comment:
I tried excluding this as well as setting this to `None` and neither seemed to work